modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/PlaybackManager.kt

```diff
@@ -2457,7 +2538,31 @@ open class PlaybackManager @Inject constructor(
         this.notificationPermissionChecker = notificationPermissionChecker
     }
 
-    fun isSleepAfterEpisodeEnabled(): Boolean = sleepAfterEpisode != 0
+    fun isSleepAfterEpisodeEnabled(): Boolean = episodesUntilSleep != 0
+
+    fun isSleepAfterChapterEnabled(): Boolean = chaptersUntilSleep != 0
+
+    private fun getCurrentChapterUuidForSleepTime(playbackState: PlaybackState): String? {
+        val currentChapter = playbackState.chapters.getChapter(playbackState.positionMs.milliseconds)
+
+        return currentChapter?.let { it.title + it.startTime }
+    }
+
+    private fun MutableMap<String, String?>.getlastListenedEpisodeUuid(): String? {
+        return this["episodeUui"]
+    }
+
+    private fun MutableMap<String, String?>.getlastListenedChapterUuid(): String? {
+        return this["chapterUui"]
```

Comment:
Looks like typo: 

Can we rename the following keys wherever they are used?
`episodeUui` -> `episodeUuid`
`chapterUui` -> `chapterUuid`

Reply:
Nice catch! Fixed here: [6006443](https://github.com/Automattic/pocket-casts-android/pull/2115/commits/600644324df63f505b8451df51d3e7fd7d60875b)